internal/pkg/build/assemblers/assembler_sandbox.go

```diff
@@ -28,8 +31,15 @@ func (a *SandboxAssembler) Assemble(b *types.Bundle, path string) (err error) {
 	if _, err := os.Stat(path); err == nil {
 		os.RemoveAll(path)
 	}
+
 	var stderr bytes.Buffer
 	cmd := exec.Command("mv", b.Rootfs(), path)
+	// If the assemble is associated to a specific image cache, we make
+	// sure that the environment variable points to it for the process
+	// that will create.
```

Comment:
I think this is outside the functionality of the assembler. All it should really be doing is taking the root file system created during the build process and package it into whatever format is specified. It shouldn't need to interact with the image cache